addons/mail/static/src/js/text_emojis_fields.js

```diff
@@ -138,6 +138,7 @@ var FieldCharEmojis = basicFields.FieldChar.extend(MailEmojisMixin, FieldEmojiCo
 registry.add('text_emojis', FieldTextEmojis);
 registry.add('char_emojis', FieldCharEmojis);
 
-return {FieldTextEmojis: FieldTextEmojis, FieldCharEmojis: FieldCharEmojis};
+// The "FieldEmojiCommon" will be used to add behavior to other widgets
```

Comment:
You can remove this comment.
It's a best practice to return all created widgets/mixins anyway.